pkgs/development/libraries/isl/0.20.0.nix

```diff
@@ -10,6 +10,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ gmp ];
 
+  configureFlags = [
+    "--with-gcc-arch=generic" # don't guess -march=/mtune=
+  ];
+
   enableParallelBuilding = true;
 
   meta = {
```

Comment:
The package has no maintainer and maybe you want to add your self here but this is not required for this PR.

Reply:
I'm quite unsure if I'm in fact the appropriate maintainer here. I'd rather not :)